opal/resource.go

```diff
@@ -416,14 +411,38 @@ func resourceResourceRead(ctx context.Context, d *schema.ResourceData, m any) di
 	}
 	d.Set("reviewer_stage", reviewerStagesI)
 
+	if resource.Metadata != nil {
```

Comment:
why do we need the `remote_info` during the read operation? Other resources types currently don't require it, right?

Reply:
this is so that we can set the `remote_info` with AWS account ID and ARN when importing from `terraformer`, so that we can support such a workflow like this:
1. Use Opal to auto-import provisioned to get permission sets into our system
2. Use `terraformer` to import accounts and permission sets into `.tf` files, with the `remote_info` filled so that the accounts can be updated (felt a bit weird to me that `remote_info` only exists when we create new remote resources)
3. From there on, set app to manual import (to be added) and then use TF to manage permission sets

I imagine we can set this for other AWS Orgs resources too, but I just did permission sets for now